src/GraphQL/Resolver/HotspotType.php

```diff
@@ -34,13 +36,36 @@ class HotspotType
      */
     public function resolveImage($value = null, $args = [], $context, ResolveInfo $resolveInfo = null)
     {
+        if ($args && $args['defaultLanguage']) {
+            $localeService = \Pimcore::getContainer()->get('pimcore.locale');
```

Comment:
please get rid of Pimcore::getContainer()

see https://github.com/pimcore/data-hub/issues/7 to get an idea